Sources/Observer.swift

```diff
@@ -124,6 +125,12 @@ extension Signal {
 	}
 }
 
+extension Signal.Observer: BindingTargetProvider {
+	public var bindingTarget: BindingTarget<Value> {
+		return BindingTarget(lifetime: lifetime, action: send)
```

Comment:
I think this is implicitly making a strong reference to `self` here. Might need a weak reference to ensure the binding doesn't keep the observer alive?

Reply:
I am not quite convinced, because usually there is an owner whose lifetime the binding target can use.

Reply:
Are you saying another lifetime should be passed in when initialising the observer?

Reply:
`Observer`s are generally owned by someone, and IMO it should be that someone who exposes the binding target for the pipe.

We could have a convenience that takes a lifetime to create a binding target though.

Reply:
This is strongly captured `self` by the way, which goes against the contract of `<~`.

Reply:
I reverted the commit that fixes that in order to un-hide the discussion that was going on. Will change it again if/when we decide how to proceed with this PR.